tests/sys_fido2_ctap/Makefile

```
@@ -1,37 +1,18 @@
-BOARD ?= nrf52840dk
-#BOARD ?= nrf52840dongle
-
 include ../Makefile.tests_common
 
-USEMODULE += fido2_ctap_transport_hid
-USEPKG += fido2_tests
+# same as CTAP_STACKSIZE
+CFLAGS += -DTHREAD_STACKSIZE_MAIN=15000
+
+# Add unittest framework
+USEMODULE += embunit
 
-USB_VID ?= $(USB_VID_TESTING)
-USB_PID ?= $(USB_PID_TESTING)
+USEMODULE += fido2_ctap
 
 # Disable user presence tests
-# Should be used when running fido2-test to make them run quicker
-#CFLAGS += -DCONFIG_FIDO2_CTAP_DISABLE_UP=1
+CFLAGS += -DCONFIG_FIDO2_CTAP_DISABLE_UP=1
 
 # Disable user LED animation
-#CFLAGS += -DCONFIG_FIDO2_CTAP_DISABLE_LED=1
-
-# FIDO2 tests except for the ones requiring user presence
-#
-# Use env -i because fido2-test has a depedency (pyscard) that needs to be
-# compiled natively (x86-64). Therefore we need to clear the flags set by e.g.
-# BOARD = nrf52840dk
-fido2-test:
-	env -i PATH=$(PATH) $(MAKE) -C $(RIOTBASE)/build/pkg/fido2_tests
-
-# FIDO2 user presence tests.
-#
-# Make sure to enable user presence tests by uncommenting CFLAGS += -DCONFIG_FIDO2_CTAP_DISABLE_UP=1
-#
-# Use env -i because fido2-test has a depedency (pyscard) that needs to be
-# compiled natively (x86-64). Therefore we need to clear the flags set by e.g.
-# BOARD = nrf52840dk
-fido2-test-up:
-	env -i PATH=$(PATH) $(MAKE) -C $(RIOTBASE)/build/pkg/fido2_tests up-tests
+CFLAGS += -DCONFIG_FIDO2_CTAP_DISABLE_LED=1
 
 include $(RIOTBASE)/Makefile.include
+
```

Whitespace fix (GitHub renders this weirdly, but I think it understands it to remove the trailing empty line):
```suggestion

include $(RIOTBASE)/Makefile.include
```